docs/docs/adrs/adr-013-equivocation-slashing.md

```diff
@@ -0,0 +1,225 @@
+goes through all undelegations and [checks](https://github.com/cosmos/cosmos-sdk/blob/bf249162e42ef084668b942e12538cb30f9e4846/x/staking/keeper/slash.go#L236)
+on whether the undelegation started before or after the infraction. If the undelegation started before the `infractionHeight`
+this undelegation is **not** slashed, otherwise the undelegation is [slashed](https://github.com/cosmos/cosmos-sdk/blob/bf249162e42ef084668b942e12538cb30f9e4846/x/staking/keeper/slash.go#L262) by
+`slashFactor`.
+
+The slashing of redelegations happens in a similar way, meaning that `Slash` goes through all redelegations and checks on whether
+the redelegation started before or after the `infractionHeight`.
+
+We believe that one of the ideas behind using `infractionHeight` to decide on whether to slash an undelegation or redelegation
+has to do with the fact that we want to slash only delegators whose voting power _contributed_ to the infraction.
+However, this is a rather obscure idea because a delegator `D` could start unbonding at height `H` and then a validator
+could intentionally perform an equivocation in the past (before `H`). In such a case, delegator `D` would still get slashed, even though `D`
+did not contribute any voting power per se for the equivocation. Furthermore, this principle is not respected in general
+(see "Slashing delegations" below).
+
+#### Slashing delegations
+Besides undelegations and redelegations, we need to slash simple delegations on the validator.
+This is performed by [deducting the appropriate amount of tokens](https://github.com/cosmos/cosmos-sdk/blob/5ca405ae067e7d8df98699f675e060f70a549976/x/staking/keeper/slash.go#L165)
+from the validator. Note that this deduction is computed based on the voting `power` the misbehaving validator
+had at the height of the equivocation. As a result of the tokens deduction, 
+the [tokens per share](https://docs.cosmos.network/v0.46/modules/staking/01_state.html#delegator-shares)
+reduce and hence later on, when delegators undelegate or redelegate, the delegators retrieve back less
+tokens, effectively having their tokens slashed. This approach of slashing delegations does not utilize the
+`infractionHeight` in any way and hence the following could occur:
+  1. a validator `V` performs an equivocation at a height `Hi`
+  2. a new delegator `D` delegates to `V` after height `Hi`
+  3. we receive the evidence of the equivocation by validator `V`
+  4. we slash the tokens of delegator `D`
+
+In the above scenario, delegator `D` is slashed, even though `D`'s voting power did not contribute to the infraction. 
+
+
+#### Old evidence
+In the single-chain case, we never act on old evidence (e.g., from 3 years ago). This is achieved through
+[CometBFT](https://docs.cometbft.com/v0.37/spec/consensus/evidence) that filters old evidence based on the parameters
```

Comment:
This check is also done in the evidence module , see https://github.com/cosmos/cosmos-sdk/blob/eb853117a97cb07ec1b6d487ad24f7a14488a23e/x/evidence/keeper/infraction.go#L69

Reply:
Added a sentence that evidence is rechecked if it's old at the application (evidence module) level.